command/sub8_missions/sub8/sub_singleton.py

```diff
@@ -312,6 +312,38 @@ def _run_look(self, spotings_req):
 
             yield self.sub.nh.sleep(.5)
 
+class PoseSequenceCommander(object):
```

Comment:
I would prefer if this was refactored to calculate all the poses in the constructor and give the user control over when to run them. This would make it easy to plug into the Searcher class we have or other custom uses.
So something like:
```
def __init__(self, sub, positions, orientations):
    # Calculate poses into self.poses

def move(self, **kwargs):
   for p in pose:
        p.go(**kwargs)

```